content/posts/2023-08-24-pycon-tw-2023-sponsors.md

```diff
@@ -0,0 +1,54 @@
+Title: PyCon Taiwan 2023 贊助商行前招呼
+Date: 2023-08-24 23:00
+Modified: 2023-08-24 23:00
+Category: sponsors
+Tags: zh-tw, en
+Slug: pycon-tw-2023-sponsors
+Authors: Kevin
+Summary: 2023 年白金級與金級贊助商在大會前向會眾打招呼！
+
```

Comment:
not sure whether you would like to add a `[TOC]` here. not required.

Reply:
Thanks, added.